MakeHist/HTLT.C

```diff
@@ -98,4 +128,100 @@ void HTLT(){
 
     leg_HTLT->Draw();
 
+    c1->SaveAs("./hists/"+prefix+"_HTLT.png");
+
+
+    //make bkg for siginificance plot
```

Comment:
Please create a new histogram, something like `totalBkg`or so instead of reusing existing diboson one.